### PR TITLE
update for the documentation

### PR DESCRIPTION
**Pull request by abrin**
culling through our change logs to try and make sure we update the documentation where we changed things:
- world-ready event
- getContentState / setContentState
- changes to choice handling and event(s)

### CODE REVIEW

File: docs/examples/10-handling-choice.md [hunk 1]
```diff
@@ -4,90 +4,150 @@ sidebar_position: 10
...
-You can then render UI for offering the choices to the user - and react to that choice by telling Canvas Panel to update the scene to reflect changes. Your UI component might be _bound_ to Canvas Panel.
-
-You might also support blending: "Show the choice with id xxx at 80% opacity AND show the choice with id yyy at 50% opacity, all the others are at 0%".
-
-Usually, if Choice is present at all, it's only one set of choices, for the whole canvas. But it's possible for a scene to be made of multiple content resources each of which have their own set of choices.
+import { GitHubDiscussion } from "../../GitHubDiscussion.js"; import
+choicesReactSandbox from '@site/sandboxes/choices-react.csb/\_load'; import
+simpleChoice from '@site/sandboxes/10-choices/simpleChoice.csb/\_load'; import
+choice1 from '@site/sandboxes/10-choices/choice1.csb/\_load'; import { Sandbox }
+from '@site/Sandbox';
```
Comment (stephenwf) [inline docs/examples/10-handling-choice.md:11]: I also ran into this formatting issue. I think we need to rename the file to be `.mdx` to fix the formatter.

File: docs/examples/10-handling-choice.md [hunk 2]
```diff
@@ -4,90 +4,150 @@ sidebar_position: 10
...
+of choices. You don't need to specify which set of choices, in the rare event
+that there is more than one - although you can specify more than one value:
 
 ```html
-<canvas-panel iiif-content="http://example.org/canvas-1.json" 
-   choice-id="http://example.org/choice-set-a/3, http://example.org/choice-set-b/7" />
+<canvas-panel
+  iiif-content="http://example.org/canvas-1.json"
+  choice-id="http://example.org/choice-set-a/3, http://example.org/choice-set-b/7"
+/>
```
Comment (stephenwf) [inline docs/examples/10-handling-choice.md:68]: A piece of tech debt is to go through and replace all self-closing tags. While it works in Vue and React, self-closing won't work in standard HTML. Doesn't have to be addressed in these files - will do it in #237

File: docs/examples/17-reacting-to-the-user.md [hunk 3]
```diff
@@ -2,26 +2,62 @@
...
+Additionally there is a 3rd event: `zoom` . The problem is that when the zoom-to
+event is fired, the viewport has only just started it's transition. It's very
+much a notification that "recently a zoom was initiated" more than a precise
+tracker. For this reason the zoom event.
 
-This will fire once the current transitioning zoom has completed. It contains information on the max/min zoom and booleans for if we have reached the max/min (isMax, isMin). The zoom event also fires when you call zoom-to . It does not fire when a user creates a custom transition.
+This will fire once the current transitioning zoom has completed. It contains
+information on the max/min zoom and booleans for if we have reached the max/min
+(isMax, isMin). The zoom event also fires when you call zoom-to . It does not
+fire when a user creates a custom transition.
```
Comment (stephenwf) [inline docs/examples/17-reacting-to-the-user.md:60]: That's a good point to add here. I think there is an example of setting up a custom transition somewhere (and also the `goToRegion` I guess!)

Comment (abrin) [reply docs/examples/17-reacting-to-the-user.md:60]: 😆 that's actually your note, it just got reformatted :(

Comment (stephenwf) [reply docs/examples/17-reacting-to-the-user.md]: Aha yeah that's the one!

**Comment by abrin** [comment]
I _think_ this one should be ready too

**Pull request merged**